packages/neutrino-lint-base/src/index.js

```diff
@@ -47,7 +47,7 @@ module.exports = neutrino => {
     options.extends = options.baseConfig.extends;
     options.useEslintrc = true;
     options.env = options.envs.reduce((env, key) => Object.assign(env, { [key]: true }, {}));
-    options.globals = options.globals.reduce((globals, key) => Object.assign(env, { [key]: true }, {}));
+    options.globals = options.globals.reduce((global, key) => Object.assign(global, { [key]: true }, {}));
```

Comment:
Just noticed that the paren for Object.assign is in the wrong spot. I'm going to create a quick patch and incorporate your changes.